test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -231,6 +230,13 @@ def get_supported_operators(cls) -> List[OperatorConfig]:
             aten_graph=True,
         )
         m = prepare_pt2e_quantizer(m, BackendAQuantizer())
+        for node in m.graph.nodes:
```

Comment:
can we check this in a different way.. it feels like this is exposing too much implementation detail, maybe just check the number of observers inserted?

Reply:
Thanks for the suggestion and changed.

Reply:
@jerryzh168  Could you take a look of this PR again? The previous comments have been adjusted.